packages/utils/src/common/pick-from-object.ts

```diff
@@ -0,0 +1,35 @@
+import { isObject } from "./is-object"
+
+export function pickFromObject(path: string, object: Record<any, any>): any {
```

Comment:
suggestion: pickValueFromObject

Reply:
```suggestion
export function pickValueFromObject(
  path: string,
  object: Record<any, any>
): unknown {
  const segments = path.split(".")
  let result: any = undefined

  for (const segment of segments) {
    const segmentsLeft = [...segments].splice(1, segments.length - 1)
    const segmentOutput = object[segment]

    if (segmentsLeft.length === 0) {
      result = segmentOutput
      break
    }

    if (isObject(segmentOutput)) {
      result = pickValueFromObject(segmentsLeft.join("."), segmentOutput)
      break
    }

    if (Array.isArray(segmentOutput)) {
      result = segmentOutput
        .map((segmentOutput_) =>
          pickValueFromObject(segmentsLeft.join("."), segmentOutput_)
        )
        .flat()
      break
    }

    result = segmentOutput
  }

  return result
}
```

Reply:
what do you think, in my opinion, it feels easier to follow

Reply:
Looks good, changed.